docs/overview-bicop.dox

```diff
@@ -40,8 +41,12 @@ sub-namespace `bicop_families`:
 (`gaussian`, `clayton`, `gumbel`, `frank`, and `joe`)
 - `two_par` contains the parametric families with two parameters
 (`student`, `bb1`, `bb6`, `bb7`, and `bb8`)
+(`gaussian`, `clayton`, `gumbel`, `frank`, and `joe`)
```

Comment:
?

Reply:
not sure what happend there, will be removed